ocaml/xapi-cli-server/cli_frontend.ml

```diff
@@ -695,6 +695,17 @@ let rec cmdtable_data : (string * cmd_spec) list =
             Cli_operations.host_emergency_reset_server_certificate
       ; flags= [Neverforward]
       } )
+  ; ( "host-reset-server-certificate"
+    , {
+        reqd= ["host-uuid"]
```

Comment:
Better use `flags= [Host_selectors]` rather than adding an explicit `host-uuid` param. See other commands for examples.